src/client/store/project.js

```diff
@@ -483,7 +547,11 @@ export const actions = {
   },
   saveProject({ state, commit }) {
     const { title } = state.settings.general
-    const blob = new Blob([JSON.stringify(state, null, 2)], { type: 'application/json' })
+    let savedState = cloneDeep(state)
+    // Reset heatstresslayers while exporting, because the layers in the geoserver
+    // are removed every day.
+    savedState.map.heatstressLayers = []
```

Comment:
I think we should actually delete the `heatstressLayers` property. It will throw an error if the property exists and the project validator is not aware of that.

To make it worse, it wil only fail after the project has been saved _again_. Very nasty bug there